### PR TITLE
Migrated to latest pluto release

### LINKED ISSUE #7363: Hook pluto extension lifecycle
### User Story

As a developer, I want my extension lifecycle to be respected.

Note that ephemeral extensions (`ephemeral?` boolean in `meta` section) should not be installed nor persisted.

Blocked by https://github.com/status-im/pluto/issues/13

### Description

*Type*: Feature

### PR DESCRIPTION
**Pull request by jeluard**
fixes #7363 

### Summary:

Introduces some new extensions syntax from latest pluto.

_NOTE to testers_ It would make sense to validate with all existing extensions listed [here](https://status.im/extensions/examples.html)

status: ready

### CODE REVIEW

**Review by cammellos — APPROVED** [review]

File: deps.edn [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {:paths ["components/src" "src" "react-native/src/cljsjs" "resources"]
  :deps  {org.clojure/clojure         {:mvn/version "1.9.0"} ;; Keep in sync with .TOOLVERSIONS
-         org.clojure/clojurescript   {:mvn/version "1.10.439"}
+         org.clojure/clojurescript   {:mvn/version "1.10.238"}
```
Comment (flexsurfer) [inline deps.edn:3]: why downgrade?

File: src/status_im/ui/screens/extensions/add/views.cljs [hunk 2]
```diff
@@ -45,25 +39,25 @@
               (i18n/label :t/extensions-disclaimer)]]
             [cartouche {:header (i18n/label :t/identifier)}
              [react/text {:style styles/text}
-              (str (get-in data ['meta :name]))]]
+              (str (get-in data [:meta :name]))]]
```
Comment (flexsurfer) [inline src/status_im/ui/screens/extensions/add/views.cljs:42]: is it still `meta` symbol in extension? or you decided to change symbols on keywords in edn?

**Review by flexsurfer — APPROVED** [review]

File: deps.edn (hunk 1 above)
Comment (jeluard) [reply deps.edn:3]: Somehow we used 2 different versions and there are some issues with using the more recent version.

File: src/status_im/ui/screens/extensions/add/views.cljs (hunk 2 above)
Comment (jeluard) [reply src/status_im/ui/screens/extensions/add/views.cljs:42]: In the parsed extension it is keywords. The extension syntax didn't change.

**Comment by Serhy** [comment]
@jeluard 
1) Wallet related extensions could be installed but they are not shown in Wallet properties after that. 
**To reproduce:**
* Open Status and create new account
* Open any of the https://status.im/extensions/play.html?hash=QmYnUj7v3UiP6X1YfRuhea5mXpjTGG1KKnCwFS4TKzKTpD or https://status.im/extensions/play.html?hash=QmXEmvhNSvAabQ8t7WX4Fz8SJ3KNh8mtLtK3985sYccjmA pages in external browser
* Add extention via QR code scan
* Once success -> navigate to `Wallet` -> top-right menu button to check whether installed extention is present 

It's all fine with nightly develop build

Seems that's only issue with this PR. 
I understand as it mentioned on https://status.im/extensions/examples.html that 'Most of those extensions are not working', but just in case, one issue with "Aragon DAO Voting Plugin" PR (I can't confirm the same extension on develop build because it ends up with `missing-property :on-success` upon install). 
**Steps to reproduce:**
* Open Status and create new account
* Install "Aragon DAO voting plugin" ( https://status.im/extensions/play.html?hash=QmXzaJygxtbYu4ttJ2WpyczuKUrYahiuNT6VV79Dswhi89 )
* Navigate to 1-1 chat and send `/get-events` / `get-events 1` message
**Actual result:**  `null is not an object... ` error appears.
Screenshot 2019-03-18 at 17 25 27

**Comment by jeluard** [comment]
@Serhy Thanks! I fixed the wallet issue.
I also fixed the Aragon extension (updated in the examples page).

There is still some issue to be fixed but I will address it in a separate PR.

**Comment by Serhy** [comment]
Looks good to me now. 
Thanks for confirming second issue above (https://github.com/status-im/status-react/pull/7740#issuecomment-473960143) is Dapp specific.
Failed autotest should be fixed in the scope of https://github.com/status-im/status-react/issues/7750 (manually all good, - I used ETHro from another account to verify recovery phrase on Wallet screen)

**Pull request merged**